Runner/JobManager.py

```diff
@@ -0,0 +1,332 @@
+from azure.storage.blob.models import BlobBlock, ContainerPermissions, ContentSettings
+from pathlib import Path
+import azure.batch.models as batchmodels
+import traceback
+import asyncio
+import Utils
+import os
+import datetime
+import functools
+import datetime
+import time
+import CustomTemplateFactory as ctm
+"""
+This module is responsible for creating, submitting and monitoring the pools and jobs
+
+"""
+
+
+_time = str(datetime.datetime.now().day) + "-" + \
+    str(datetime.datetime.now().hour) + "-" + \
+    str(datetime.datetime.now().minute)
+
+
+async def submit_job(batch_service_client, template, parameters):
```

Comment:
📓 Please think about adding simple`Unit Test` for key functionality.

Reply:
Haven't done this yet. But i'll do after we move over to the new repo.